distro/configuration/globalproperties/globalproperties-core_demo.xml

```diff
@@ -5,4 +5,30 @@
             <value>50</value>
         </globalProperty>
     </globalProperties>
+
+    <!-- Queue Configuration -->
+
+    <!-- https://app.openconceptlab.org/#/orgs/openmrs/sources/DemoQueueConcepts/concepts/DemoQueue-Service/ -->
+    <globalProperty>
+        <property>queue.serviceConceptSetName</property>
+        <value>a8f3f64a-11d5-4a09-b0fb-c8118fa349f3</value>
+    </globalProperty>
+
+    <!-- https://app.openconceptlab.org/#/orgs/openmrs/sources/DemoQueueConcepts/concepts/DemoQueue-Priority/ -->
+    <globalProperty>
+        <property>queue.priorityConceptSetName</property>
+        <value>78063dec-b6d8-40c1-9483-dd4d3c3ca434</value>
+    </globalProperty>
+
+    <!-- https://app.openconceptlab.org/#/orgs/openmrs/sources/DemoQueueConcepts/concepts/DemoQueue-Status/ -->
+    <globalProperty>
+        <property>queue.statusConceptSetName</property>
+        <value>d60ffa60-fca6-4c60-aea9-a79469ae65c7</value> <!-- PIH Queue Status Set -->
```

Comment:
Should probably get rid of the comment mentioning PIH here.

Reply:
Oh, good catch - removed @ibacher